docs/reference/api/generated-api-html/index.html

```diff
@@ -2410,6 +2410,7 @@ <h3 class="field-label">Example data</h3>
       "option" : "basic"
     },
     "dbt" : {
+      "gitRepoBranch" : "gitRepoBranch",
```

Comment:
Huh I thought I pushed a change that stopped the generation of api docs.

Reply:
Why? People do use our generated API docs. We should try to keep them generating?